recipes/beauty/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conans import ConanFile, tools
+from conan.tools.cmake import CMake
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.43.0"
+
+
+class BeautyConan(ConanFile):
+    name = "beauty"
+    homepage = "https://github.com/dfleury2/beauty"
+    description = "HTTP Server above Boost.Beast"
+    topics = ("http", "server", "boost.beast")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "CMakeDeps", "CMakeToolchain"
+
+    requires = ("boost/[>1.70.0]@",
+                "openssl/1.1.1o@")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "8",
+            "clang": "11",
+            "Visual Studio": "16",
+        }
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "17")
```

Comment:
```suggestion
            tools.check_min_cppstd(self, "20")
```

beauty seems to require C++20.
https://github.com/dfleury2/beauty/blob/master/CMakeLists.txt#L25

Reply:
Done.
I will fix the CMakeLists.txt on next release